index.html

```diff
@@ -266,137 +289,324 @@ <h4 class="packcage-title">Package Summery Details</h4>
                 </li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Glass bottom boat trips</span>
+                  <span class="hightlight-text">Glass bottom boat trips</span>
                 </li>
               </ul>
             </div>
           </div>
           <div class="row details-box-2">
             <div class="col">
-              <ul class="details-box__list">
-                <li class="details-box__title">Meal Plan Available</li>
+              <ul class="details-box-list">
+                <li class="details-box-title">Meal Plan Available</li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Half Board</span>
+                  <span class="hightlight-text">Half Board</span>
                 </li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">All Inclusive</span>
+                  <span class="hightlight-text">All Inclusive</span>
                 </li>
               </ul>
             </div>
             <div class="col">
-              <ul class="details-box__list">
-                <li class="details-box__title">Entertainment</li>
+              <ul class="details-box-list">
+                <li class="details-box-title">Entertainment</li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Live music</span>
+                  <span class="hightlight-text">Live music</span>
                 </li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">DJ & Mauritian Dance Show</span>
+                  <span class="hightlight-text">DJ & Mauritian Dance Show</span>
                 </li>
               </ul>
             </div>
             <div class="col">
-              <ul class="details-box__list">
-                <li class="details-box__title">Check In & Check Out</li>
+              <ul class="details-box-list">
+                <li class="details-box-title">Check In & Check Out</li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Check In Time 14:00</span>
+                  <span class="hightlight-text">Check In Time 14:00</span>
                 </li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Check Out Time 11:00</span>
+                  <span class="hightlight-text">Check Out Time 11:00</span>
                 </li>
               </ul>
             </div>
           </div>
-
         </div>
     </section>
+    <section class="other_section">
```

Comment:
Is it according to BEM? Could you please rename this class? Maybe `packages-section`?

Reply:
> Is it according to BEM? Could you please rename this class? Maybe `packages-section`?
did it